DevDeployment/src/Modules/MockWebApp/main.tf

```diff
@@ -0,0 +1,25 @@
+resource "azurerm_windows_web_app" "mock_webapp_service" {
+  name                = var.name
+  location            = var.location
+  resource_group_name = var.resource_group_name
+  service_plan_id     = var.service_plan_id
+  tags                = var.tags
+
+  site_config {
```

Comment:
This section seems to have inconsistent indentation. Please review and format nicely.

Reply:
Hi,
Fixed the indentation.

Reply:
Lines 9 to 11 are only indented by one space, should be two I think. Also the empty line 15 can be removed.

Reply:
Fixed the indentation and removed empty line number 15.